tests/integration_tests/extract_nn_data.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import os
+import pickle
+
+import cv2
+import depthai as dai
+from utils import extract_main_slug, extract_parser, get_input_shape
+
+argparser = argparse.ArgumentParser()
+argparser.add_argument("-ip", help="IP address of the device", default="")
+argparser.add_argument(
+    "-img", "--img_path", help="Path to the input image", required=True, type=str
```

Comment:
Do we have the test images also stored together with the outputs the models generated? It would be good to have them so we can reuse them maybe in the future.

Reply:
Yes, it is saved along with the outputs in the `extract_nn_data.py` [here](https://github.com/luxonis/depthai-nodes/blob/c99bbc4e7173167646afee00f4d2202400c53fb3/tests/integration_tests/extract_nn_data.py#L104).